cli.sh

```diff
@@ -59,10 +60,12 @@ case $1 in
     rename "$2"
     ;;
   password)
+    checkrpi
     checkroot
     password "$2"
     ;;
   sshkey)
+    checkrpi
```

Comment:
we wanna change only on root when nonrpi system,
but not "/home/pi/.ssh/authorized_keys"